ckanext/harvest/helpers.py

```diff
@@ -157,3 +156,16 @@ def bootstrap_version():
             'bs2' if
             p.toolkit.config.get('ckan.base_public_folder') == 'public-bs2'
             else 'bs3')
+
+
+def get_latest_job(harvest_id):
+    context = {'model': model, 'session': model.Session, 'user': p.toolkit.c.user or p.toolkit.c.author}
+    try:
+        hs = p.toolkit.get_action('harvest_source_show')(context, {'id': harvest_id})
+        last_job = hs.get('status', {}).get('last_job', {})
+        if not last_job:
+            return {}
+        job = p.toolkit.get_action('harvest_job_show')(context, {'id': last_job.get('id')})
```

Comment:
The `harvest_job_show` action will only work for logged in users. The `harvest_source_show` action will work for everyone.
Add a exception for not authorized.

Reply:
done